app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -54,6 +55,15 @@ class PatientSummaryScreenControllerTest {
 
   private val uiEvents = PublishSubject.create<UiEvent>()
   private val reporter = MockReporter()
+
+  // If the placeholder count is changed, the `params for placeholder bp items`
+  // method will also need to be changed to account for the change in the
+  // number of placeholder.
+
+  // TODO: Change the method to generate test inputs dynamically based on
+  // the number of placeholders declared.
```

Comment:
This looks easily doable. Let's do it as part of this commit?

Reply:
Discussed: Is not very easily doable. Will revisit in the future.